finetuner-workflow/finetune-workflow.yaml

```diff
@@ -194,9 +198,13 @@ spec:
     - name: dataset_downloader_tag
       value: 'cd6408a'
     - name: finetuner_image
-      value: 'gooseai/finetuner'
+      value: 'docker.io/harubaru1/finetuner'
```

Comment:
Should we package these images in `ml-containers` instead of using public domains like `docker.io`?

Reply:
Yes, these are just placeholders

Reply:
After the PR gets merged I'm thinking about making a PR against `ml-containers` to build a finetuner image